tools/calyx-pass/manual.md

```diff
@@ -0,0 +1,44 @@
+# User Manual
```

Comment:
Merge this into the `README.md`?

Reply:
I considered, but felt it would make it too cluttered.